atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -471,6 +471,7 @@ def add_log_type(output):
         digests = self.get_digests()
         repositories = self.get_repositories(digests)
         arch = os.uname()[4]
+        tags = [image.tag for image in self.workflow.tag_conf.primary_images]
```

Comment:
Should this be a `set` to avoid possible duplicates? Probably not a problem now, but if we ever use multiple pulp servers, there could be a duplicate for each tag.